langchain/src/storage/tests/vercel_kv.int.test.ts

```diff
@@ -7,58 +7,100 @@ import { VercelKVStore } from "../vercel_kv.js";
 import { createDocumentStoreFromByteStore } from "../encoder_backed.js";
 import { Document } from "../../document.js";
 
-test("VercelKVStore", async () => {
-  const store = new VercelKVStore({
-    client: createClient({
-      url: process.env.VERCEL_KV_API_URL!,
-      token: process.env.VERCEL_KV_API_TOKEN!,
-    }),
-  });
-  const value1 = new Date().toISOString();
-  const value2 = new Date().toISOString() + new Date().toISOString();
-  const encoder = new TextEncoder();
-  await store.mset([
-    ["key1", encoder.encode(value1)],
-    ["key2", encoder.encode(value2)],
-  ]);
-  const retrievedValues = await store.mget(["key1", "key2"]);
-  expect(retrievedValues).toEqual([
-    encoder.encode(value1),
-    encoder.encode(value2),
-  ]);
-  for await (const key of store.yieldKeys()) {
-    console.log(key);
+const getClient = () => {
+  if (!process.env.VERCEL_KV_API_URL || !process.env.VERCEL_KV_API_TOKEN) {
+    throw new Error(
+      "VERCEL_KV_API_URL and VERCEL_KV_API_TOKEN must be set in the environment"
+    );
   }
-  await store.mdelete(["key1", "key2"]);
-  const retrievedValues2 = await store.mget(["key1", "key2"]);
-  expect(retrievedValues2).toEqual([undefined, undefined]);
-});
+  const client = createClient({
+    url: process.env.VERCEL_KV_API_URL,
+    token: process.env.VERCEL_KV_API_TOKEN,
+  });
+  return client;
+};
 
-test("Encoder-backed", async () => {
-  const store = createDocumentStoreFromByteStore(
-    new VercelKVStore({
-      client: createClient({
-        url: process.env.VERCEL_KV_API_URL!,
-        token: process.env.VERCEL_KV_API_TOKEN!,
-      }),
-    })
-  );
-  const value1 = new Date().toISOString();
-  const value2 = new Date().toISOString() + new Date().toISOString();
-  const [doc1, doc2] = [
-    new Document({ pageContent: value1 }),
-    new Document({ pageContent: value2 }),
-  ];
-  await store.mset([
-    ["key1", doc1],
-    ["key2", doc2],
-  ]);
-  const retrievedValues = await store.mget(["key1", "key2"]);
-  expect(retrievedValues).toEqual([doc1, doc2]);
-  for await (const key of store.yieldKeys()) {
-    console.log(key);
-  }
-  await store.mdelete(["key1", "key2"]);
-  const retrievedValues2 = await store.mget(["key1", "key2"]);
-  expect(retrievedValues2).toEqual([undefined, undefined]);
+describe("VercelKVStore", () => {
+  const client = getClient();
+
+  afterEach(async () => await client.flushall());
+
+  test("VercelKVStore can preform all operations", async () => {
```

Comment:
This old test is unchanged, except it's now inside this describe block & the client isn't initialized inside the test